activities/human-resources/job-description.md

```diff
@@ -0,0 +1,51 @@
+---
+type: Resources
+---
+
+# [Job title]
+
+Come help producers of open source software for public use – such as in cities and other public organizations – develop and maintain software and policy together. The Foundation for Public Code provides review of codebases that are in our stewardship to help developers build trusted and reliable software that can be widely reused and foster a strong community around it.
+
+We test against the Standard for Public Code to make codebases understandable, reusable and community based. Our standards are high, as well as opinionated, to ensure codebases can be trusted as the infrastructure of society and improve continually, and policy makers can keep technology and architectural choices open.
+
+Part of this effort is to [bespoke text contextualising role in broader foundation work].
+
+## What the [JOB TITLE] does
+
+As a [JOB TITLE] you work together with the community to [bespoke text]. This means you will:
+[bespoke text starting with a verb]
+
+[CALL TO ACTION - bespoke text e.g. So, if you enjoy working together with a wide array of developers to make open source codebases really shine as well as improve the role of technology in society, this job is for you.]
+
+## What we expect from all of our team members
+
+You should:
+* share our core values: openness, quality, trust, community, helpfulness
+* understand the value of and be experienced with open source products and communities
+* want to work for the public good
+* have a collaborative work attitude and take pride in making things together
+* be able to work independently
+* be a strong communicator in English
+* have international experience
+* be an active participant in the codebase community you support
+
+## What we expect from the [JOB TITLE]
```

Comment:
Is the purpose of this uppercasing that the text needs to be uppercased?

Reply:
We're missing a convention on how customisable content (or instructions) are highlighted on templates. Since neither \```\ or [] is semantically correct, do you have any suggestions?

Reply:
[text here] seems sufficiently clear.
Or else: write a 'reading aid' at the top to explain?

Reply:
made an issue for this

Reply:
I think `[` and `]` are fine to use. I was just wondering whether the uppercase text for these was meant to be replaced with uppercase or not.